kingpin/utils.py

```diff
@@ -347,6 +347,30 @@ def convert_json_to_dict(json_file, tokens):
     return decoded
 
 
+def order_dict(obj):
```

Comment:
Ok well either way -- would you like to rename this? I don't really have a strong opinion here, its just a quick helper method.